### PR TITLE
RD-6846 Allow explicit scripts in script_runner

### PR DESCRIPTION
**Pull request by tehasdf**
- not only specialcase `implementation: script/path.py`, but also a script written right there
- set those operations to executor=auto, which means it'll be either host_agent, or CDA, depending on whether it is contained_in a Compute or not

### CODE REVIEW

File: dsl_parser/elements/operation.py [hunk 1]
```diff
@@ -415,3 +428,27 @@ def process_operation(
                     'workflow' if is_workflows else 'operation'))
         error_message = base_error_message + partial_error_message
         raise exceptions.DSLParsingLogicException(error_code, error_message)
+
+
+def _is_explicit_script(script):
```
Comment (geokala) [inline dsl_parser/elements/operation.py:433]: Leaving aside the implementation of this (which I'm still considering and we're also discussing out of band), I'd probably call this _is_inline_script

File: dsl_parser/tests/test_parser_api.py [hunk 2]
```diff
@@ -2108,6 +2108,46 @@ def assert_operation(op, extra_properties=False):
         self.assertEqual(workflow2['parameters']['key']['default'], 'value')
         self.assertEqual(workflow['plugin'], constants.SCRIPT_PLUGIN_NAME)
 
+    def test_script_mapping_explicit_script(self):
```
Comment (geokala) [inline dsl_parser/tests/test_parser_api.py:2111]: We should probably have a test that covers a single line inline script too?

Comment (tehasdf) [reply dsl_parser/tests/test_parser_api.py:2111]: This test already has one! `op3: "ctx logger info 'hello world'"`

Comment (geokala) [reply dsl_parser/tests/test_parser_api.py:2111]: So it does- nice!

**Review by geokala — APPROVED** [review]

File: dsl_parser/elements/operation.py [hunk 3]
```diff
@@ -446,18 +445,25 @@ def _is_inline_script(script):
...
+
+    if '/' in script or '\\' in script:
+        # script is one line and contains a slash/backslash - interpret
+        # this as a path (posix or windows).
+        # This means it's impossible to have 1-line scripts containing
+        # slashes, but this stays backwards-compatible
+        return False
+
+    if (
+        script.count('.') > 0 and
```
Comment (geokala) [inline dsl_parser/elements/operation.py:468]: Isn't this less efficient than "'.' in script"?

**Review by geokala — APPROVED** [review]

**Pull request merged**